.prettierrc.json

```diff
@@ -2,10 +2,9 @@
   "printWidth": 80,
   "tabWidth": 2,
   "useTabs": false,
-  "semi": false,
```

Comment:
Forcing the use of semicolons is fine, but forcing not to use semicolons is rude. To minimize unexpected bugs, you should absolutely avoid using these **FANCY** options.